fera_full.py

```
@@ -1,3 +1,189 @@
+                        indexador_fera.createNewDbFile(global_settings.root) 
+                        sqliteconn = utilities_general.connectDB(str(global_settings.pathdb))                           
+                        tupleinfo = indexador_fera.addrels('relatorio', pathpdfinput = notindexed, pathdbext=sys.argv[1], rootx=global_settings.root, sqliteconnx=sqliteconn)                            
+                        indexing = True
+                #indexador_fera.gather_information_fromdb()
+                if(tocommit):
+                    sqliteconn.commit()
+                utilities_general.initiate_indexing_thread()
+                global_settings.initiate_processes()
+                start_time = time.time()
+                fera.start_fera_app() 
+            elif(".db" == extension.lower()):
```

Ruff

Yoda condition detected

Rewrite as `extension.lower() == ".db"`

(SIM300)